.circleci/config.yml

```diff
@@ -131,7 +131,11 @@ executors:
     machine:
       # This custom CircleCi Ubuntu image has a pre-installed Docker, so we are not using the CircleCI remote Docker when we
       # initiate this machine. More here: https://discuss.circleci.com/t/linux-machine-executor-images-2022-january-q1-update/42831
-      image: ubuntu-2004:2023.07.1
+      image: ubuntu-2004:2023.10.1
+  ubuntu_vm_xlarge:
```

Comment:
@leoporoli do we hav an idea of the extra cost this will incur?

Reply:
@adschwartz 
based on this info: https://circleci.com/product/features/resource-classes/
it will cost `100 credits/min` and the actual "large" costs `20 credits/min` so yeap probably is not worth it to activate it now, I'll downgrade it and see if the other changes help without this